simulator/Field.py

```diff
@@ -1,21 +1,58 @@
-#!/usr/bin/env python3
```

Comment:
We need to keep this comment at the top of the file because it sets the interpreter with which the file is executed.